config/config.exs

```diff
@@ -1,6 +1,6 @@
 # This file is responsible for configuring your application
 # and its dependencies with the aid of the Mix.Config module.
-use Mix.Config
+import Config
```

Comment:
I need to check but I think currently the Travis build matrix still includes old Elixir version/s that doesn't work with this change.